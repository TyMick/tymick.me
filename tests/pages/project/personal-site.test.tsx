import React from "react";
import renderer from "react-test-renderer";
import PersonalSite from "../../../pages/project/personal-site";

describe("PersonalSite", () => {
  it("matches snapshot", async () => {
    expect(renderer.create(<PersonalSite />).toJSON()).toMatchSnapshot();
  });
});
