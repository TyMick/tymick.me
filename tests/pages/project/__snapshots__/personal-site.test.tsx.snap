// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalSite matches snapshot 1`] = `
<div
  className="cap-width-lg container"
>
  <h1>
    About 
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      TyMick.me
    </a>
  </h1>
  <p>
    Why yes, I built this site! I built 
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      TyMick.me
    </a>
     with
     
    <a
      href="https://nextjs.org/"
    >
      Next.js
    </a>
     and host it on
     
    <a
      href="https://vercel.com/"
    >
      Vercel
    </a>
    . You can view the
     
    <a
      href="https://github.com/TyMick/tymick.me"
    >
      source
    </a>
     on GitHub.
  </p>
  <p>
    Design I accomplished with
     
    <a
      href="https://getbootstrap.com/"
    >
      Bootstrap
    </a>
     (via
     
    <a
      href="https://react-bootstrap.netlify.com/"
    >
      React Bootstrap
    </a>
    ) and a little custom 
    <a
      href="https://sass-lang.com/"
    >
      Sass
    </a>
    . Just wanted something minimal.
  </p>
</div>
`;
